includes/degree-functions.php

```diff
@@ -145,3 +145,31 @@ function ucf_tuition_fees_degree_layout( $resident, $nonresident ) {
 <?php
 	return ob_get_clean();
 }
+
+/** Degree Search Typeahead Functions */
+function main_site_degree_search_display( $output ) {
+	ob_start();
+?>
+	<div class="input-group degree-search">
+		<input type="text" class="form-control degree-search-typeahead" placeholder="Search for degree programs">
+		<span class="input-group-btn">
+			<button id="ucf-degree-search-submit" type="submit" class="btn btn-primary"><span class="fa fa-search"></span></button>
```

Comment:
Submit button needs some screenreader-friendly text, with either `aria-label="Search Degrees"` on the button or with an inner `<span class="sr-only">Search Degrees</span>`.

Adding `aria-hidden="true"` to the .fa-search icon would also be beneficial.